qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -656,7 +656,13 @@ def set_mark(self, key):
             key: mark identifier; capital indicates a global mark
         """
         # strip the fragment as it may interfere with scrolling
-        url = self.current_url().adjusted(QUrl.RemoveFragment)
+        try:
+            url = self.current_url().adjusted(QUrl.RemoveFragment)
+        except:
```

Comment:
Never use a bare `except:`, that'd catch any exception (even a typo somewhere, for example). This should be `except qtutils.QtValueError:` instead.